README.md

```diff
@@ -8,11 +8,11 @@ Creating test fixtures should be easy.
 
 ## Example
 
-Pass a zod schema to the `createFixture` method.
+Pass a zod schema to the `generate` method.
```

Comment:
A little context to this:

I'm partial to generator because I wanted the flexibility to create things that weren't just fixtures. An example:

```ts
import { z, ZodString } from 'Zod';
import { Generator, generate } from 'zod-fixture';

const FixtureStringGenerator = Generator({
  schema: ZodString,
  match: () => true,
  generator: () => 'abc'
})

const SerializeStringGenerator = Generator({
  schema: ZodString,
  match: () => true,
  generator: () => ({ type: string })
})

const schema = z.string();

// in this case, we get a fixture
generate(schema, {
  extend: [FixtureStringGenerator ]
})

// but in this case, we get a serialized version
generate(schema, {
  extend: [SerializeStringGenerator ]
}) // 
```

What about having both? `createFixture` could be preconfigured to generate fixtures and `generate` could be a blank template (ie no preconfigured generators).

Reply:
I see, my first thought is that I like the proposed idea.
Let me thinker on this over the weekend.

Reply:
I refactored so that the base level exports are:
- createFixture: a core instance preconfigured with generators to spit out fixtures
- Core: the base class for running generators
- Generator: for creating custom generators

I like the idea of having preconfigured instances (createFixture, createSerialized, createAdjustable, etc).

Reply:
Love it!